boards/tsi/ek_tsi_skyp/ek_tsi_skyp_defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 TSI
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_RUNTIME_NMI=y
+CONFIG_ARM_TRUSTZONE_M=n
+CONFIG_ARM_MPU=y
+CONFIG_QEMU_ICOUNT_SHIFT=7
+
+# GPIOs
+CONFIG_GPIO=y
+
+# Serial
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+
+# Build a Secure firmware image
+CONFIG_TRUSTED_EXECUTION_SECURE=n
```

Comment:
I have added more configs, you may want to pull those in add well. Especially to automatically create IMAGE0_testfile.hex.

Reply:
I have already taken care of the changes related to the auto scripts, so that file will not be affected.